domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Controllers/TransferAgreementsControllerTest.cs

```diff
@@ -308,13 +320,14 @@ public async Task GetBySubjectId_ShouldReturnTransferAgreements_WhenUserHasTrans
                     SenderId = Guid.Parse(sub),
                     SenderName = "Producent A/S",
                     SenderTin = senderTin,
-                    ReceiverTin = "87654321"
+                    ReceiverTin = "87654321",
+                    ReceiverReference = Guid.NewGuid()
                 }
             });
 
-        var authenticatedClient = factory.CreateAuthenticatedClient(sub, tin: receiverTin);
+        var client = factory.CreateAuthenticatedClient(sub, tin: receiverTin);
```

Comment:
I've noticed that in almost all test you are creating a "senderClient" with a random sender guid.
Why not create a field variable so you don't have to instantiate a new client on every test when there is no need to specific values.

Reply:
I have done what you asked, and only created new clients where relevant.